components/DropdownMenu.tsx

```diff
@@ -8,27 +8,30 @@ interface DropDownProps {
   orderBySort: (orderby: string) => void;
 }
 
+type SelectedOption = "recent" | "like";
+
 export default function DropdownMenu({ orderBySort }: DropDownProps) {
   const [isOpen, setIsOpen] = useState(false);
-  const [selectedOption, setSelectedOption] = useState<string>("최신순");
+  const [selectedOption, setSelectedOption] =
+    useState<SelectedOption>("recent");
 
   const toggleDropdown = () => {
     setIsOpen(!isOpen);
   };
 
-  const handleOptionClick = (value: string) => {
+  const handleOptionClick = (value: "recent" | "like") => {
```

Comment:
```suggestion
  const handleOptionClick = (value: SelectedOption) => {
```

이런곳에 같은 타입으로 사용해주시면 됩니다